### PR TITLE
[ADD] CRM: Forecast report

### PR DESCRIPTION
**Pull request by jero-odoo**
Docs task: https://www.odoo.com/mail/view?model=project.task&res_id=3961797&access_token=508f9a3c-592c-4908-992d-65948a83e7c6
Adding new doc to outline CRM: Forecast report

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hey @larm-odoo  this is ready for review. Thanks!

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (larm-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:61]: wht a brilliant workaround

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (larm-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:89]: I know we are supposed to use the icon name, but this really says 'Graph' and that is more clear to me. I woudl argue for the use of "graph" but of course, defer to SALI. I think we need to has this out in a writers talk since thisocmes up a lot, especially when the icon description is bizarre.

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (larm-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:102]: This is a good doc to link to if you see it fts: https://www.odoo.com/documentation/17.0/applications/essentials/search.html

**Review by larm-odoo — APPROVED** [review]
Hi @jero-odoo - great doc, I understood and followed along even though it's totlaly out of my wheelhouse! I only had a handful of suggestions, so approving now.

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (jero-odoo) [reply content/applications/sales/crm/performance/forecast_report.rst:89]: I've spoken to Sam about this, we are going to use the official names for now. There are a few icons that have names that may not match directly with the action on the screen that in the future I could see changing the term for (like the save icon, since the official name is "cloud upload" I believe) but for now we are supposed to use the dev name for consistency.

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (jero-odoo) [reply content/applications/sales/crm/performance/forecast_report.rst:102]: This doc has a lot for filters and groupings, but not specifically measures, which is what the tip is directing to

**Comment by jero-odoo** [comment]
@ksc-odoo This is ready for final review. Thanks!

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:34]:
```suggestion
this date directly from the :guilabel:`Forecast` page, select the Kanban card for the desired opportunity, then click and drag the card to the
```
SUGGESTION: for added specificity and enhanced clarity

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:43]:
```suggestion
After an opportunity is added to a new month, the *Expected Closing* field on the opportunity form is updated to
```
SUGGESTION: added specificity to fall in line with the same reasoning as above re: the addition of "opportunity form" to the explanation. (using italics because that's a field name that isn't being shown directly on the 'Forecast' page -- it's only present on the Opportunity form)

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:49]:
```suggestion
   The *Expected Closing* field can also be manually updated on the opportunity card. To do that, click on the Kanban card for
   an opportunity on the :guilabel:`Forecast` page to open the opportunity's detail form. Click in the :guilabel:`Expected Closing`
   field, and use the calendar popover to select a new closing date.
```

Comment (ksc-odoo) [reply content/applications/sales/crm/performance/forecast_report.rst:49]: OPTIONAL: adjusted formatting (when necessary) and added specificity to the steps for enhanced clarity

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:54]:
```suggestion
At the top of the column for each month on the :guilabel:`Forecast` reporting page, to the right of the progress bar, is a sum of the prorated
```
OPTIONAL: just a helpful reminder to the reader, so they can always make sure they're on the right page

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:69]:
```suggestion
   The first opportunity, `Global Solutions`, has an expected revenue of `$3,800`, and a probability
```
SUGEGSTION: added commas for additional sentence clarity and enhanced flow

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:72]:
```suggestion
   The second opportunity, `Quote for 600 Chairs`, has an expected revenue of `$22,500`, and a
```

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:89]: I was going to leave a very similar suggestion here as @larm-odoo -- but since you've already spoken to Sam about it -- you can just keep it as-is 👍

Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:89]:
```suggestion
Click the :icon:`fa-area-chart` :guilabel:`(area chart)` icon to change to graph view. Then, click the
```

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:98]:
```suggestion
Click the |pivot| to change to the pivot view, or the |list| to change to the list view.
```
no italics (nor capitalization) necessary here. it's my understanding that the only view options that we change at all are Kanban and Gantt -- and even then, it's just capitalized. feel free to double-check with sam for clarification, if you'd like 👍

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:101]: I would also suggest linking to a Reporting view specific doc to let the reader quickly access more detailed information about this kind of view -- maybe to just a specific anchor? Either way, it's something to consider...

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (ksc-odoo) [inline content/applications/sales/crm/performance/forecast_report.rst:109]:
```suggestion
   To save this (or any) report configuration in the *Favorites* column of the :guilabel:`Search...` bar drop-down menu of filter and grouping options, see :ref:`search/favorites`.
```
OPTIONAL: additional specification to enhanced the clarity of this explanation...

**Comment by jero-odoo** [comment]
@samueljlieber this is ready for you. Thanks!

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (samueljlieber) [inline content/applications/sales/crm/performance/forecast_report.rst:25]: Just to stay consistent with how the other icons and views are presented in this doc
```suggestion
assigned expected closing date. The opportunities are grouped by month in a :icon:`oi-view-kanban`
:guilabel:`(Kanban)` view.
```

File: content/applications/sales/crm/performance/forecast_report.rst
Comment (samueljlieber) [inline content/applications/sales/crm/performance/forecast_report.rst:18]: In my opinion we could do without this image, just to reduce the number of overall images :)
```suggestion
```

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo, nice work on this brand new CRM reporting doc on the Forecast report! Nice use of the math directive and solid example showing how prorated revenue works 👏

Approving with a super small nitpick! Thanks for your hard work!
.....
@robodoo delegate=jero-odoo

**Comment by jero-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
